sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpSender.cs

```diff
@@ -361,106 +361,105 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
         /// <summary>
         ///
         /// </summary>
-        /// <param name="message"></param>
+        /// <param name="messages"></param>
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
-        public override async Task<long> ScheduleMessageAsync(
-            ServiceBusMessage message,
+        public override async Task<long[]> ScheduleMessagesAsync(
+            IList<ServiceBusMessage> messages,
             CancellationToken cancellationToken = default)
         {
-            long sequenceNumber = 0;
-            Task scheduleTask = _retryPolicy.RunOperation(async (timeout) =>
+            long[] seqNumbers = null;
+            await _retryPolicy.RunOperation(async (timeout) =>
             {
-                sequenceNumber = await ScheduleMessageInternalAsync(
-                    message,
+                seqNumbers = await ScheduleMessageInternalAsync(
+                    messages,
                     timeout,
                     cancellationToken).ConfigureAwait(false);
             },
             _connectionScope,
-            cancellationToken);
-            await scheduleTask.ConfigureAwait(false);
-            return sequenceNumber;
+            cancellationToken).ConfigureAwait(false);
+            return seqNumbers ?? Array.Empty<long>();
         }
 
         /// <summary>
         ///
         /// </summary>
-        /// <param name="message"></param>
+        /// <param name="messages"></param>
         /// <param name="timeout"></param>
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
```

Comment:
Are these doc comments supposed to be empty?

Reply:
Will do a final pass on these internal members before GA.